fateme/challenge-1.js

```diff
@@ -17,8 +17,12 @@ function isPrime(n) {
   return true;
 }
 
-// test case for prime numbers
+function isTypeValid(value, trueType) {
+  if (typeof value === trueType) return true;
```

Comment:
You don't need an `if - else` statement for this. You can use the following shorthand:
`return typeof value === trueType`